src/pages/workspace/expensifyCard/issueNew/CardNameStep.tsx

```diff
@@ -26,10 +25,6 @@ function CardNameStep() {
     const [issueNewCard] = useOnyx(ONYXKEYS.ISSUE_NEW_EXPENSIFY_CARD);
 
     const isEditing = issueNewCard?.isEditing;
-    const data = issueNewCard?.data;
-
-    const userName = PersonalDetailsUtils.getUserNameByEmail(data?.assigneeEmail ?? '', 'firstName');
-    const defaultCardTitle = `${userName}'s Card`;
```

Comment:
@parasharrajat why did you remove this logic?

Reply:
https://github.com/Expensify/App/issues/48732 I know this issue, but shouldn't we have the default name for the physical card?

Reply:
yes, it's for both physical and virtual

Reply:
you should check for `issueNewCard?.data?.cardType` if it's CONST.EXPENSIFY_CARD.CARD_TYPE.PHYSICAL should be "[userName]'s Card" like it's implemented now

am I right @mountiny?

Reply:
Yeah only physical cards should have the suggested name per the GH